src/app/pregnancies/page.tsx

```
@@ -1,17 +1,64 @@
+            <Link href="/dashboard">
+              <Button variant="outline" size="sm">Go to dashboard</Button>
+            </Link>
+          </div>
+        </div>
+
+        <div className="rounded-3xl border border-[#E8F6F5] bg-gradient-to-br from-blue-50 to-blue-100 p-6 shadow-sm">
+          <div className="flex items-start gap-4">
+            <Calendar className="mt-0.5 size-6 text-blue-600" />
+            <div>
+              <p className="font-semibold text-blue-900">Ready to start tracking?</p>
+              <p className="mt-1 text-sm text-blue-800">Once your clinic connects your pregnancy record, you'll see detailed tracking and progress here.</p>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)